lib/GameConstants.h

```diff
@@ -756,7 +756,7 @@ class Obj
 		ARTIFACT = 5,
 		PANDORAS_BOX = 6,
 		BLACK_MARKET = 7,
-		BOAT = 8,
+		TRANSPORT = 8,

```

Comment:
But why? Everyone knows what Boat is, but "Transport" is unclear to say at least.

Reply:
That one would be Airship or skyship...
So Boat (aka Ship) could be VEHICLE or CRAFT in generic form and Ship in particular cases when you want to refer to water vehicles.

Reply:
Maybe only renaming (without functional changes) could be in separate PR if we agreed on something. It's a lot of stuff to rename, so I'm with @DjWarmonger here